manage_plugin_page.php

```diff
@@ -135,7 +135,7 @@ function ( $p_p1, $p_p2 ) {
 		$t_name = '<a href="' . string_attribute( plugin_page( $t_page, false, $t_basename ) ) . '">' . $t_name . '</a>';
 	}
 
-	if( !is_blank( $t_author ) ) {
+	if( !empty( $t_author ) ) {
```

Comment:
If you want I can change that to `!empty( trim( $t_author ) )` to have exactly the same behavior with strings as with an is_blank() call.